modules/ai/assets/js/editor/pages/form-layout/index.js

```diff
@@ -86,6 +102,13 @@ const FormLayout = ( { onClose, onInsert, onGeneration, onSelect, DialogHeaderPr
 		lastRun.current();
 
 		setIsPromptEditable( false );
+		setCurrentPage( 1 );
+	};
+
+	const handleRegenerate = () => {
```

Comment:
if you have `handleRegenerate` maybe change the `handleSubmit` to `handleGenerate`?

Reply:
Done.